core/src/main/java/io/questdb/client/Sender.java

```diff
@@ -90,6 +93,62 @@ static LineSenderBuilder builder() {
         return new LineSenderBuilder();
     }
 
+    /**
+     * Create a Sender instance from a configuration string.
+     * <br>
+     * Configuration string fully describes Sender configuration.
+     * <p>
+     * <b>Example 1</b><br>
+     * This example creates a Sender instance that connects to a QuestDB server over HTTP transport. The created Sender
+     * will auto-flush data when number of buffered rows reaches 1000.
+     * <code>http::addr=localhost:9000;auto_flush_rows=1000;</code>
+     * <br>
+     * <b>Example 2</b><br>
+     * This example creates a Sender instance that connects to a QuestDB server over TCP transport.
+     * <code>tcp::addr=localhost:9009;</code>
+     * <p>
+     * Refer to <a href="https://questdb.io/docs/">QuestDB documentation</a> for a full list of configuration options.
```

Comment:
Would be good to have a URL for these options here in the src, even if the page doesn't quite exist yet.

Reply:
Risk of it getting unmaintained.
A central documentation location is preferred.
As a compromise you might just point to the clients doc page: We'll put a link to the config strings there for sure once done.